LEGO1/lego/legoomni/src/police/police.cpp

```diff
@@ -19,3 +59,11 @@ MxLong Police::Notify(MxParam& p_param)
 
 	return 0;
 }
+
+// FUNCTION: LEGO1 0x1005e530
+void Police::VTable0x50()
+{
+	LegoWorld::VTable0x50();
+	PlayMusic(0xc);
```

Comment:
Can you use the JukeBoxScript enum rather than hardcoding a value here? `e_policeStation` would be equivalent to `0xc`